earthmover/earthmover.py

```diff
@@ -517,7 +517,7 @@ def build_package_graph(self, root_node: str, package_subgraph: Graph, packages_
             # Install packages if necessary, or retrieve path to package yaml file
             package_node = self.package_graph.nodes[package_name]
             if install:
-                installed_package_yaml = package_node['package'].install(packages_dir)
+                installed_package_yaml = package_node['package'].install(packages_dir, self.state_configs.get('git_auth_timeout', 60))
```

Comment:
This is really petty, but can we pass the `git_auth_timeout` keyword to the second argument of this method? This second argument is _technically_ optional if we set a default argument value of 60, so this will improve clarity when going through the code later.